src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -35,7 +35,8 @@ public class CertificateGenerator
 
         // This can't be too short as there might be a time skew between machines,
         // but also can't be too long, as the CRL is cached by the machine
-        private TimeSpan _crlValidityPeriod = TimeSpan.FromMinutes(2);
+        private TimeSpan _crlValidityGracePeriodStart = TimeSpan.FromMinutes(5);
+        private TimeSpan _crlValidityGracePeriodEnd = TimeSpan.FromMinutes(5);
 
         // Give the cert a grace period in case there's a time skew between machines
         private readonly TimeSpan _gracePeriod = TimeSpan.FromHours(1);
```

Comment:
For this particular case, it might not be possible as the request for the CRL is a call made by Windows or OpenSSL based on the CRL URI baked into the certificate itself (i.e., I don't see how it's modifiable per-request)

If we are keen to solve the skew problem for this purpose, we might consider, upon PUT for an Authority certificate or another well-known CertificateResource, to send the "now" of the client over, and then assuming the machine being tested won't correct its time in the meantime. 

In a real-world environment, it's likely that a skew of five minutes will cause problems anyway with various forms of security anyway (e.g., Kerberos). I arrived at five minutes because in Windows: 

> If the KDC verifies that the time it decrypts from the authenticator is within a specified amount of the local time on the KDC (the default is five minutes), the system can assume that the credentials presented are genuine.

https://technet.microsoft.com/en-us/library/cc780011%28v=ws.10%29.aspx?f=255&MSPPError=-2147217396

(there are a couple of other articles stating the same, so it should be a reasonable assumption that clocks should be within five minutes within a given environment)